services/tenant-ui/frontend/src/components/contacts/.prettierrc

```diff
@@ -0,0 +1 @@
+printWidth: 80
```

Comment:
Probably don't intend on having this .prettierrc down in a specific components folder?

Reply:
Agreed. Would you be up for having the whole repo default to 80 characters. I find anything longer harder to read.

Reply:
![Screenshot from 2022-08-23 10-45-52](https://user-images.githubusercontent.com/479074/186228661-f84cb006-52e0-4731-910b-27f8bef14e59.png)